ci/kokoro/docker/build-in-docker-bazel.sh

```diff
@@ -75,6 +81,7 @@ io::log "Fetching dependencies"
 
 echo "================================================================"
 io::log "Compiling and running unit tests"
+echo "bazel test" "${bazel_args[@]}"
```

Comment:
Debugging?

Reply:
Yes, but if that is Okay I think we should leave it.  It may be useful in the future when we are trying to understand what the build is actually doing.